cyverse-de/ui
# React update

- Update `react-gwt` to latest version
- Update to `webpack` 4.x
- Update to `React 16.7.0`
- Update `GWT` to `2.8.2`
- Update `storybook` (and related) for react to `4.1.7`
- Update `react-scripts` to latest version

And many more version updates. See `package.json` for full list.
